Changelog.md

```diff
@@ -3,6 +3,7 @@
 ## 1.2.0
 
 - Update Camel Quarkus Catalog from 3.10.0 to 3.11.0
+- Use `.camel.yaml` as output format for transforming Camel Routes

```

Comment:
it is only for the placeholder, the user can still provide whatever filename for output format